addons/html_editor/static/tests/html_field.test.js

```diff
@@ -954,83 +954,74 @@ describe("sandbox", () => {
-//     Partner._records = [
-//         {
-//             id: 1,
-//             txt: "<p class='test_target'><a href='/test'>This website</a></p>",
-//         },
-//     ];
-//     await mountView({
-//         type: "form",
-//         resId: 1,
-//         resModel: "partner",
-//         arch: `
-//             <form>
-//                 <field name="txt" widget="html"/>
-//             </form>`,
-//     });
-
-//     // Test the popover option to edit the link
-//     click(".test_target a")
-//     const a = document.querySelector(".test_target a");
-//     // Wait for the popover to appear
-//     await nextTick();
-//     a.click();
-//     await nextTick();
-//     // Click on the edit link icon
-//     document.querySelector("a.mx-1.o_we_edit_link.text-dark").click();
-//     // Make sure popover is closed
-//     await new Promise((resolve) => $(a).on("hidden.bs.popover.link_popover", resolve));
-//     let labelInputField = document.querySelector(".modal input#o_link_dialog_label_input");
-//     let linkPreview = document.querySelector(".modal a#link-preview");
-//     assert.strictEqual(
-//         labelInputField.value,
-//         "This website",
-//         "The label input field should match the link's content"
-//     );
-//     assert.strictEqual(
-//         linkPreview.innerText.replaceAll("\u200B", ""),
-//         "This website",
-//         "Link label in preview should match label input field"
-//     );
-
-//     // Click on discard
-//     await click(document, ".modal .modal-footer button.btn-secondary");
-
-//     const p = document.querySelector(".test_target");
-//     // Select link label to open the floating toolbar.
-//     setSelection(p, 0, p, 1);
-//     await nextTick();
-//     // Click on create-link button to open the Link Dialog.
-//     document.querySelector("#toolbar #create-link").click();
-//     await nextTick();
-
-//     labelInputField = document.querySelector(".modal input#o_link_dialog_label_input");
-//     linkPreview = document.querySelector(".modal a#link-preview");
-//     assert.strictEqual(
-//         labelInputField.value,
-//         "This website",
-//         "The label input field should match the link's content"
-//     );
-//     assert.strictEqual(
-//         linkPreview.innerText,
-//         "This website",
-//         "Link label in preview should match label input field"
-//     );
-
-//     // Edit link label.
-//     await editInput(labelInputField, null, "New label");
-//     assert.strictEqual(
-//         linkPreview.innerText,
-//         "New label",
-//         "Preview should be updated on label input field change"
-//     );
-//     // Click "Save".
-//     await click(document, ".modal .modal-footer button.btn-primary");
-//     assert.strictEqual(
-//         p.innerText.replaceAll("\u200B", ""),
-//         "New label",
-//         "The link's label should be updated"
-//     );
-// });
+test("link preview in Link Dialog", async () => {
```

Comment:
The test should be moved up inside the {} above, cause the resModel partner is not found on runbot